### PR TITLE
HW1 cdw2146

### PR DESCRIPTION
**Pull request by chriswebb09**

### CODE REVIEW

File: cdw2146.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Name: Christopher Webb
+
+UNI: cdw2146
+
+GitHub: [link](https://github.com/chriswebb09)
+
+GitHub README: [link](https://github.com/chriswebb09/chriswebb09/blob/main/README.md)
+
+# Project Proposal
+* Project Idea: Legit(ish) AR flight sim in iOS
```
Comment (timkpaine) [inline cdw2146.md:12]: add link to your project

File: cdw2146.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Project Idea: Browser plugin to run content of webpage through AI detection software APIs
+* Project Idea: Clean user interface that help formulate useful AI prompts 
+* Project Idea: HackerNews clone or library, that makes sorting and searching more intuitive 
+* Project Idea: Flocking behavior library for ARKit Apple 
+* Project Idea: Site to learn binary and hex interactively
+
+JS / Python / Swift / TypeScript
+
+# Project Contributions
+
```
Comment (timkpaine) [inline cdw2146.md:23]: add link to project markdowns

**Review by timkpaine — CHANGES_REQUESTED** [review]
remove empty file at `projects/c++/swift.md`

File: cdw2146.md (hunk 1 above)
Comment (chriswebb09) [reply cdw2146.md]: I updated it to an AR project I've been working on. After seeing HW2, I just want to clarify, you mean I should create new one from scratch?

**Review by timkpaine — CHANGES_REQUESTED** [review]
`projects/javascript/danjer-js.md`
`projects/swift/sourcery.md`

both of these files are empty

File: cdw2146.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* Project Idea: Legit(ish) AR flight sim in iOS [Github Repo Candidate](https://github.com/chriswebb09/ARKitDrone)
+* Project Idea: Integrate Apple Watch health data into AR navigation to create exercise app for AR
+* Project Idea: Browser plugin to run content of webpage through AI detection software APIs
+* Project Idea: Clean user interface that help formulate useful AI prompts 
+* Project Idea: HackerNews clone or library, that makes sorting and searching more intuitive 
+* Project Idea: Flocking behavior library for ARKit Apple 
+* Project Idea: Site to learn binary and hex interactively
+
+[Javascript](https://github.com/chriswebb09/project-proposals-s2023/tree/main/projects/javascript) / [Python](https://github.com/chriswebb09/project-proposals-s2023/tree/main/projects/python) / [Swift](https://github.com/chriswebb09/project-proposals-s2023/tree/main/projects/swift) / [C++](https://github.com/chriswebb09/project-proposals-s2023/tree/main/projects/c%2B%2B)
```
[the remark this answers is not in the document]
Comment (chriswebb09) [reply cdw2146.md:21]: Updated

File: cdw2146.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+# [Project Contributions](https://github.com/chriswebb09/project-proposals-s2023/tree/main/projects)
+
+* [Swift](https://github.com/apple/swift) C++ and Swift - [Project Readme](https://github.com/chriswebb09/project-proposals-s2023/blob/main/projects/swift/swift.md)
```
[the remark this answers is not in the document]
Comment (chriswebb09) [reply cdw2146.md]: Fixed

File: cdw2146.md (hunk 2 above)
Comment (chriswebb09) [reply cdw2146.md:24]: I reverted commits so you should be able to create the conflict as planned.

**Pull request closed without merging**